mirrord/layer/src/file/ops.rs

```diff
@@ -93,18 +83,6 @@ impl Drop for RemoteFile {
     }
 }
 
-/// should_ignore(path: P, write: bool)
-/// Checks if the file should be ignored, and returns a [`Bypass::IgnoredFile`] if it should.
-macro_rules! should_ignore {
```

Comment:
Hmm, my intention was to always have globals used explicitly. I found it hard to navigate through layer with all those `static`s accessed in multiple places. Not a strong opinion though, I can revert

Reply:
Does the new way of ignoring files also count as as using globals for that matter:
```Rust
crate::setup().filter_ignored_file(path.to_str().unwrap_or_default(), false)?;
```
I don't mean we should use a `FILE_FILTER` global again, I just mean, since that new way of checking is also a bit long, we could still have the macro for it.

Reply:
Okok, reasonable

Reply:
Restored the macro. Changed name to `ensure_not_ignored` (following [`anyhow::ensure`](https://docs.rs/anyhow/latest/anyhow/macro.ensure.html) convention) though - `should_ignore` sounds like its returning a value to be handled.